qcengine/programs/cfour/harvester.py

```diff
@@ -63,7 +63,7 @@ def harvest_outfile_pass(outtext):
     #          vcc/ecc
 
     # fmt: off
-    NUMBER = "((?:[-+]?\\d*\\.\\d+(?:[DdEe][-+]?\\d+)?)|(?:[-+]?\\d+\\.\\d*(?:[DdEe][-+]?\\d+)?))"
+    NUMBER = regex.NUMBER
```

Comment:
fyi that qcel has a `NUMBER` https://github.com/MolSSI/QCElemental/blob/16b2de82eb603c8d868a9c5e27a696c769f1f6ae/qcelemental/molparse/regex.py#L18-L22

Reply:
Thanks for pointing this out. It requires `re.VERBOSE` which will break some of my existing regexes. Is it possible to convert a VERBOSE `re` to a standard one?

Reply:
maybe could incorporate into the string itself with `(?aiLmsux)` from https://docs.python.org/3.8/library/re.html. I just add the `re.VERBOSE` or `re.X`

Reply:
I can certainly turn on verbose mode inline with `(?x)`, but I don't know how to turn it off.

Reply:
Nevermind, there is a `(?x:<regex>)` syntax that solves this issue.